server/api/constants.js

```diff
@@ -13,3 +13,21 @@ exports.personTypes = {
 	'5': 'יזם',
 	'6': 'אחר'
 };
+
+exports.tags = [
+	{'tagName':'דיור' , 'tagId':'1', 'functionName': 'isHousing'}
```

Comment:
When using objects, no need to add `'` for keys, no need to change but for next tine this can also work:
```
{tagName:'housing'....}
```